From b3d6e23cb433847bf7068f78476cee9762aee15c Mon Sep 17 00:00:00 2001
From: Aman-Surkar <aman_surkar@persistent.com>
Date: Thu, 28 Dec 2023 17:56:05 +0000
Subject: [PATCH] fix-build-failure-on-s390x

---
 src/normalizer.cc | 13 +++++++------
 src/normalizer.h  |  1 +
 2 files changed, 8 insertions(+), 6 deletions(-)

diff --git a/src/normalizer.cc b/src/normalizer.cc
index 2ab8084..b18bd10 100644
--- a/src/normalizer.cc
+++ b/src/normalizer.cc
@@ -264,10 +264,11 @@ std::string Normalizer::EncodePrecompiledCharsMap(
 
 #ifdef IS_BIG_ENDIAN
   uint32 *data = reinterpret_cast<uint32 *>(const_cast<char *>(blob.data()));
-  for (int i = 0; i <= trie_blob.size() / 4; ++i)
-    data[i] = util::Swap32(data[i]);
+  for (int i = 0; i <= blob.size() / 4; ++i)
+    data[i] = Swap32(data[i]);
 #endif
 
+  blob.append(normalized.data(), normalized.size());
   return blob;
 }
 
@@ -279,13 +280,12 @@ util::Status Normalizer::DecodePrecompiledCharsMap(
   if (blob.size() <= sizeof(trie_blob_size) ||
       !string_util::DecodePOD<uint32>(
           absl::string_view(blob.data(), sizeof(trie_blob_size)),
-          &trie_blob_size) ||
-      trie_blob_size >= blob.size()) {
+          &trie_blob_size)) {
     return util::InternalError("Blob for normalization rule is broken.");
   }
 
 #ifdef IS_BIG_ENDIAN
-  trie_blob_size = util::Swap32(trie_blob_size);
+  trie_blob_size = Swap32(trie_blob_size);
 #endif
 
   if (trie_blob_size >= blob.size())
@@ -294,9 +294,10 @@ util::Status Normalizer::DecodePrecompiledCharsMap(
   blob.remove_prefix(sizeof(trie_blob_size));
 
 #ifdef IS_BIG_ENDIAN
+  CHECK_OR_RETURN(buffer);
   buffer->assign(blob.data(), trie_blob_size);
   uint32 *data = reinterpret_cast<uint32 *>(const_cast<char *>(buffer->data()));
-  for (int i = 0; i < trie_blob_size / 4; ++i) data[i] = util::Swap32(data[i]);
+  for (int i = 0; i < buffer->size() / 4; ++i) data[i] = Swap32(data[i]);
   *trie_blob = absl::string_view(buffer->data(), trie_blob_size);
 #else
   *trie_blob = absl::string_view(blob.data(), trie_blob_size);
diff --git a/src/normalizer.h b/src/normalizer.h
index c79813c..37fdb8a 100644
--- a/src/normalizer.h
+++ b/src/normalizer.h
@@ -22,6 +22,7 @@
 #include <vector>
 
 #include "common.h"
+#include "util.h"
 #include "sentencepiece_model.pb.h"
 #include "sentencepiece_processor.h"
 #include "third_party/absl/strings/string_view.h"
-- 
2.34.1

