From 012c07e16419493c6d3b7d165944d6099e3c84ce Mon Sep 17 00:00:00 2001
From: Deepali Chourasia <deepch23@in.ibm.com>
Date: Thu, 11 May 2023 05:01:43 +0000
Subject: [PATCH] add path for libatomic.so to list of libs

---
 src/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 1c7726e..3b06461 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -213,7 +213,7 @@ if ((${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm") OR
   find_library(ATOMIC_LIB NAMES atomic libatomic.so libatomic.so.1)
   if (ATOMIC_LIB)
     message(STATUS "Found atomic: ${ATOMIC_LIB}")
-    list(APPEND SPM_LIBS "atomic")
+    list(APPEND SPM_LIBS ${ATOMIC_LIB})
   endif()
 endif()
 
-- 
2.34.1

