diff --git a/src/bpe_model.h b/src/bpe_model.h
index c6e1abe..8021d4e 100644
--- a/src/bpe_model.h
+++ b/src/bpe_model.h
@@ -15,8 +15,8 @@
 #ifndef BPE_MODEL_H_
 #define BPE_MODEL_H_
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "model_interface.h"
+#include "sentencepiece_model.pb.h"
 
 namespace sentencepiece {
 namespace bpe {
diff --git a/src/bpe_model_trainer.h b/src/bpe_model_trainer.h
index 051ac46..ff25342 100644
--- a/src/bpe_model_trainer.h
+++ b/src/bpe_model_trainer.h
@@ -20,7 +20,7 @@
 #include <unordered_map>
 #include <vector>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
+#include "sentencepiece_model.pb.h"
 #include "trainer_interface.h"
 
 namespace sentencepiece {
diff --git a/src/builder.h b/src/builder.h
index f0b959a..49d2884 100644
--- a/src/builder.h
+++ b/src/builder.h
@@ -19,8 +19,8 @@
 #include <string>
 #include <vector>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "common.h"
+#include "sentencepiece_model.pb.h"
 #include "sentencepiece_processor.h"
 #include "third_party/absl/strings/string_view.h"
 
diff --git a/src/char_model.h b/src/char_model.h
index 23d0016..cd32875 100644
--- a/src/char_model.h
+++ b/src/char_model.h
@@ -15,8 +15,8 @@
 #ifndef CHAR_MODEL_H_
 #define CHAR_MODEL_H_
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "model_interface.h"
+#include "sentencepiece_model.pb.h"
 
 namespace sentencepiece {
 namespace character {
diff --git a/src/char_model_trainer.h b/src/char_model_trainer.h
index f7b8a39..e563819 100644
--- a/src/char_model_trainer.h
+++ b/src/char_model_trainer.h
@@ -15,7 +15,7 @@
 #ifndef CHAR_MODEL_TRAINER_H_
 #define CHAR_MODEL_TRAINER_H_
 
-#include "builtin_pb/sentencepiece_model.pb.h"
+#include "sentencepiece_model.pb.h"
 #include "trainer_interface.h"
 
 namespace sentencepiece {
diff --git a/src/model_factory.h b/src/model_factory.h
index 0502af1..76abce7 100644
--- a/src/model_factory.h
+++ b/src/model_factory.h
@@ -17,8 +17,8 @@
 
 #include <memory>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "model_interface.h"
+#include "sentencepiece_model.pb.h"
 
 namespace sentencepiece {
 
diff --git a/src/model_interface.cc b/src/model_interface.cc
index 43dfbd1..bb4b636 100644
--- a/src/model_interface.cc
+++ b/src/model_interface.cc
@@ -14,8 +14,8 @@
 
 #include <algorithm>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "model_interface.h"
+#include "sentencepiece_model.pb.h"
 #include "third_party/absl/memory/memory.h"
 #include "third_party/absl/strings/str_format.h"
 #include "util.h"
diff --git a/src/model_interface.h b/src/model_interface.h
index 27dad99..8734367 100644
--- a/src/model_interface.h
+++ b/src/model_interface.h
@@ -22,9 +22,9 @@
 #include <utility>
 #include <vector>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "common.h"
 #include "normalizer.h"
+#include "sentencepiece_model.pb.h"
 #include "sentencepiece_processor.h"
 #include "third_party/absl/strings/string_view.h"
 #include "third_party/darts_clone/darts.h"
diff --git a/src/normalizer.h b/src/normalizer.h
index 13166ca..ab12fac 100644
--- a/src/normalizer.h
+++ b/src/normalizer.h
@@ -21,8 +21,8 @@
 #include <utility>
 #include <vector>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "common.h"
+#include "sentencepiece_model.pb.h"
 #include "sentencepiece_processor.h"
 #include "third_party/absl/strings/string_view.h"
 #include "third_party/darts_clone/darts.h"
diff --git a/src/pretokenizer_for_training.h b/src/pretokenizer_for_training.h
index 0c84a08..2d3bc82 100644
--- a/src/pretokenizer_for_training.h
+++ b/src/pretokenizer_for_training.h
@@ -18,8 +18,8 @@
 #include <memory>
 #include <string>
 
-#include "builtin_pb/sentencepiece.pb.h"
 #include "common.h"
+#include "sentencepiece.pb.h"
 #include "sentencepiece_processor.h"
 #include "third_party/absl/strings/string_view.h"
 
diff --git a/src/sentencepiece_processor.cc b/src/sentencepiece_processor.cc
index a4dd575..1e87a80 100644
--- a/src/sentencepiece_processor.cc
+++ b/src/sentencepiece_processor.cc
@@ -16,12 +16,12 @@
 #include <set>
 #include <utility>
 
-#include "builtin_pb/sentencepiece.pb.h"
 #include "common.h"
 #include "filesystem.h"
 #include "model_factory.h"
 #include "model_interface.h"
 #include "normalizer.h"
+#include "sentencepiece.pb.h"
 #include "sentencepiece_processor.h"
 #include "third_party/absl/memory/memory.h"
 #include "third_party/absl/strings/numbers.h"
diff --git a/src/sentencepiece_processor_test.cc b/src/sentencepiece_processor_test.cc
index bceba2c..2be81c1 100644
--- a/src/sentencepiece_processor_test.cc
+++ b/src/sentencepiece_processor_test.cc
@@ -16,11 +16,11 @@
 #include <utility>
 
 #include "builder.h"
-#include "builtin_pb/sentencepiece.pb.h"
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "filesystem.h"
 #include "model_interface.h"
 #include "normalizer.h"
+#include "sentencepiece.pb.h"
+#include "sentencepiece_model.pb.h"
 #include "sentencepiece_processor.h"
 #include "sentencepiece_trainer.h"
 #include "testharness.h"
diff --git a/src/sentencepiece_trainer.cc b/src/sentencepiece_trainer.cc
index e36aa9c..f8a05a1 100644
--- a/src/sentencepiece_trainer.cc
+++ b/src/sentencepiece_trainer.cc
@@ -16,10 +16,10 @@
 #include <vector>
 
 #include "builder.h"
-#include "builtin_pb/sentencepiece.pb.h"
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "common.h"
 #include "normalizer.h"
+#include "sentencepiece.pb.h"
+#include "sentencepiece_model.pb.h"
 #include "sentencepiece_trainer.h"
 #include "spec_parser.h"
 #include "third_party/absl/strings/str_cat.h"
diff --git a/src/sentencepiece_trainer_test.cc b/src/sentencepiece_trainer_test.cc
index b78b1d2..5ad66bd 100644
--- a/src/sentencepiece_trainer_test.cc
+++ b/src/sentencepiece_trainer_test.cc
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.!
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "filesystem.h"
+#include "sentencepiece_model.pb.h"
 #include "sentencepiece_trainer.h"
 #include "testharness.h"
 #include "third_party/absl/strings/str_cat.h"
diff --git a/src/spm_decode_main.cc b/src/spm_decode_main.cc
index a65e615..61699ee 100644
--- a/src/spm_decode_main.cc
+++ b/src/spm_decode_main.cc
@@ -16,10 +16,10 @@
 #include <string>
 #include <vector>
 
-#include "builtin_pb/sentencepiece.pb.h"
 #include "common.h"
 #include "filesystem.h"
 #include "flags.h"
+#include "sentencepiece.pb.h"
 #include "sentencepiece_processor.h"
 #include "third_party/absl/strings/str_split.h"
 #include "util.h"
diff --git a/src/spm_encode_main.cc b/src/spm_encode_main.cc
index 85480c6..67fd9f5 100644
--- a/src/spm_encode_main.cc
+++ b/src/spm_encode_main.cc
@@ -17,10 +17,10 @@
 #include <unordered_map>
 #include <vector>
 
-#include "builtin_pb/sentencepiece.pb.h"
 #include "common.h"
 #include "filesystem.h"
 #include "flags.h"
+#include "sentencepiece.pb.h"
 #include "sentencepiece_processor.h"
 #include "third_party/absl/strings/str_cat.h"
 #include "third_party/absl/strings/str_join.h"
diff --git a/src/spm_export_vocab_main.cc b/src/spm_export_vocab_main.cc
index 729faf2..5245365 100644
--- a/src/spm_export_vocab_main.cc
+++ b/src/spm_export_vocab_main.cc
@@ -15,10 +15,10 @@
 
 #include <sstream>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "common.h"
 #include "filesystem.h"
 #include "flags.h"
+#include "sentencepiece_model.pb.h"
 #include "sentencepiece_processor.h"
 
 DEFINE_string(output, "", "Output filename");
diff --git a/src/spm_normalize_main.cc b/src/spm_normalize_main.cc
index 4b6f5bc..ebcf488 100644
--- a/src/spm_normalize_main.cc
+++ b/src/spm_normalize_main.cc
@@ -13,12 +13,12 @@
 // limitations under the License.!
 
 #include "builder.h"
-#include "builtin_pb/sentencepiece.pb.h"
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "common.h"
 #include "filesystem.h"
 #include "flags.h"
 #include "normalizer.h"
+#include "sentencepiece.pb.h"
+#include "sentencepiece_model.pb.h"
 #include "sentencepiece_processor.h"
 #include "sentencepiece_trainer.h"
 
diff --git a/src/spm_train_main.cc b/src/spm_train_main.cc
index a2ec3a7..b516c37 100644
--- a/src/spm_train_main.cc
+++ b/src/spm_train_main.cc
@@ -14,8 +14,8 @@
 
 #include <map>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "flags.h"
+#include "sentencepiece_model.pb.h"
 #include "sentencepiece_trainer.h"
 #include "third_party/absl/strings/ascii.h"
 #include "third_party/absl/strings/str_split.h"
diff --git a/src/trainer_factory.h b/src/trainer_factory.h
index d563f7d..a11cbc0 100644
--- a/src/trainer_factory.h
+++ b/src/trainer_factory.h
@@ -17,7 +17,7 @@
 
 #include <memory>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
+#include "sentencepiece_model.pb.h"
 #include "trainer_interface.h"
 
 namespace sentencepiece {
diff --git a/src/trainer_interface.h b/src/trainer_interface.h
index 552b206..e17a732 100644
--- a/src/trainer_interface.h
+++ b/src/trainer_interface.h
@@ -23,9 +23,9 @@
 #include <utility>
 #include <vector>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "common.h"
 #include "filesystem.h"
+#include "sentencepiece_model.pb.h"
 #include "sentencepiece_processor.h"
 #include "sentencepiece_trainer.h"
 #include "util.h"
diff --git a/src/unigram_model.h b/src/unigram_model.h
index df84260..2f66a5f 100644
--- a/src/unigram_model.h
+++ b/src/unigram_model.h
@@ -20,10 +20,10 @@
 #include <utility>
 #include <vector>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "common.h"
 #include "freelist.h"
 #include "model_interface.h"
+#include "sentencepiece_model.pb.h"
 #include "third_party/darts_clone/darts.h"
 
 namespace sentencepiece {
diff --git a/src/unigram_model_test.cc b/src/unigram_model_test.cc
index e8ea0c6..dacec38 100644
--- a/src/unigram_model_test.cc
+++ b/src/unigram_model_test.cc
@@ -17,7 +17,7 @@
 #include <string>
 #include <vector>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
+#include "sentencepiece_model.pb.h"
 #include "sentencepiece_processor.h"
 #include "testharness.h"
 #include "third_party/absl/strings/str_cat.h"
diff --git a/src/unigram_model_trainer.h b/src/unigram_model_trainer.h
index a0c1cea..91fbeb4 100644
--- a/src/unigram_model_trainer.h
+++ b/src/unigram_model_trainer.h
@@ -20,7 +20,7 @@
 #include <utility>
 #include <vector>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
+#include "sentencepiece_model.pb.h"
 #include "third_party/absl/strings/string_view.h"
 #include "trainer_interface.h"
 #include "unigram_model.h"
diff --git a/src/unigram_model_trainer_test.cc b/src/unigram_model_trainer_test.cc
index f15d7f9..7a813cd 100644
--- a/src/unigram_model_trainer_test.cc
+++ b/src/unigram_model_trainer_test.cc
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.!
 
-#include "builtin_pb/sentencepiece_model.pb.h"
+#include "sentencepiece_model.pb.h"
 #include "sentencepiece_processor.h"
 #include "sentencepiece_trainer.h"
 #include "testharness.h"
diff --git a/src/word_model.h b/src/word_model.h
index 0048478..34470f9 100644
--- a/src/word_model.h
+++ b/src/word_model.h
@@ -15,8 +15,8 @@
 #ifndef WORD_MODEL_H_
 #define WORD_MODEL_H_
 
-#include "builtin_pb/sentencepiece_model.pb.h"
 #include "model_interface.h"
+#include "sentencepiece_model.pb.h"
 
 namespace sentencepiece {
 namespace word {
diff --git a/src/word_model_test.cc b/src/word_model_test.cc
index 01c174c..aefb174 100644
--- a/src/word_model_test.cc
+++ b/src/word_model_test.cc
@@ -14,7 +14,7 @@
 
 #include <string>
 
-#include "builtin_pb/sentencepiece_model.pb.h"
+#include "sentencepiece_model.pb.h"
 #include "testharness.h"
 #include "util.h"
 #include "word_model.h"
diff --git a/src/word_model_trainer.h b/src/word_model_trainer.h
index 44aa657..76f8f32 100644
--- a/src/word_model_trainer.h
+++ b/src/word_model_trainer.h
@@ -15,7 +15,7 @@
 #ifndef WORD_MODEL_TRAINER_H_
 #define WORD_MODEL_TRAINER_H_
 
-#include "builtin_pb/sentencepiece_model.pb.h"
+#include "sentencepiece_model.pb.h"
 #include "trainer_interface.h"
 
 namespace sentencepiece {
